modules/drivers/presto-jdbc/test/metabase/driver/presto_jdbc_test.clj

```diff
@@ -194,5 +194,9 @@
 
 (deftest test-database-connection-test
   (mt/test-driver :presto-jdbc
-    (testing "can-test-database-connection works"
-      (is (nil? (database-api/test-database-connection :presto-jdbc (:details (mt/db))))))))
+    (testing "can-test-database-connection works properly"
+      ;; for whatever reason, :let-user-control-scheduling is the only "always available" option that goes into details
+      ;; the others (ex: :auto_run_queries and :refingerprint) are one level up (fields in the model, not in the details
+      ;; JSON blob)
+      (let [db-details (assoc (:details (mt/db)) :let-user-control-scheduling false)]
```

Comment:
`:let-user-control-scheduling` is just a marker that the schedules for the two syncs are system-randomized and not user input. It is optional and i imagine can be safely ignored. None of the databases in my dev copy have this key and it shouldn't be important enough here i don't think

```sql
select distinct json_object_keys(details::json) from metabase_database;
  json_object_keys
--------------------
 password
 db
 host
 port
 user
 additional-options
 tunnel-enabled
 ssl
 dbname
(9 rows)
```

Reply:
oh i guess i misread. this key is added as one of the "bad" keys that presto blows up on. I see.

Reply:
The point of the test, though, is to ensure that even if a user happens to set it, the connection still succeeds (see the screenshot [here](https://github.com/metabase/metabase/issues/17199#issuecomment-891368881)).